README.md

```diff
@@ -93,7 +93,7 @@ Content is in `content/en/docs`. Make your changes to the desired file.
 
 Use the `git status` command at any time to see what files you've changed.
 
-If you have installed [Hugo](https://gohugo.io/getting-started/installing/) and want to preview your changes locally, run from the repo root:
+If you have installed [Hugo](https://gohugo.io/getting-started/installing/) (version `0.71.1` or higher) and want to preview your changes locally, run from the repo root:
```

Comment:
Saying "version xxx or higher" can lead to issues if the higher version that what is specified in `netlify.toml` introduces quirks we're not aware of.    I prefer not hard-coding a version number at all, since it's quite possible the README won't get updated when we update the Hugo version in `netlify.toml`.   
Since line 5 states to use the Hugo version specified in `netlify.toml`, what about saying "version specified in `netlify.toml` here as well?

Reply:
Contributors to Armory docs _should_ install the same version of Hugo if they want to compile and run locally. The other option for contributors is to use the Deploy Preview that's generated with each PR to preview their changes.

The theme is only a small part of our docs site. The majority of functionality comes from Hugo itself.  Additionally, we created shortcodes for functionality we need that's not included with Hugo or the theme.  In my experience contributing to projects that use Hugo, backwards or forwards compatibility with one's project is not guaranteed.

Homebrew supports installing multiple versions of the same product and enables you to switch between versions with the `brew switch` command.

Reply:
> Homebrew supports installing multiple versions of the same product and enables you to switch between versions with the `brew switch` command.

I didn't know that! I'll update the branch shortly.

Reply:
Quick update; I don't know if this is Linux specific or not, but I can't find a way to install or switch between multiple version of a package. There's lots of info out there about `brew list` and `brew list`, I don't know if it's deprecated or just not implemented for 'nix.